src/java/util/serialized/MaterialSet.java

```diff
@@ -0,0 +1,21 @@
+package util.serialized;
+
+import java.util.List;
+
+// used in disk operations to be converted into a material later

```

Comment:
Take a look at JavaDoc when commenting on the function, parameters, return types or exceptions of classes or functions.
`/**
*/`